src/test/java/io/devnoob/marble/api/MarbleControllerTest.java

```diff
@@ -166,10 +173,16 @@ void getMarblesByUserIdReturnCorrectResponseBody() throws Exception {
             assertEquals(expectedJsonResponse, actualJsonResponse); 
             Mockito.verify(marbleRepository, times(1)).getMarblesByUserId(1L);
         }
+    }
 
+    @Nested
+    @DisplayName("Test get latest marbles by given user_id API")
+    class TestGetLatestMarblesByUserId {
         @Test
-        @DisplayName("Should return OK and in the form of DESC when user exist")
+        @DisplayName("Should return OK when user exists")
```

Comment:
This test case should not rely on the "the method should return the maximum number of marbles when the limit exceeds the total number" feature since the feature should also be tested.